bfex/components/scraper/scraper.py

```diff
@@ -0,0 +1,25 @@
+from bs4 import BeautifulSoup
+from abc import ABC
+import requests
+import json
+import re
+from bfex.components.scraper.scrapp import *
+
+base_url = "https://www.ualberta.ca/science/about-us/contact-us/faculty-directory/"
+
+class Scraper(ABC):
+    
+    def __init__(self, url, type):
+        self.url = url
+        self.type = type
+        
+    def validate_url(self):
+        pass
+
+    def get_url(self):
```

Comment:
Sorry one more comment :(

this isn't getting the url right? its getting the content?

I think we should make @GarryBullock  change his uml cause `get_content` would return the content of the html and `get_scrapp` should return the srapp. He has it different in uml

https://github.com/CMPUT401-Find-an-Expert/Find-an-Expert/wiki/Technical-Documentation